### ISSUE TITLE
`deploy html` outside of working directory fails 

### ISSUE THREAD

**Issue by kgartland-rstudio** [rstudio/rsconnect-python#359]
Deploying html from outside of the directory fails with the following error:

```
> rsconnect deploy html bundles/html-single-file/index.html -n dogfood
Validating server... 	[OK]
Validating app mode... 	[OK]
Making bundle ... 	[ERROR]: Unable to include the file index.html in the bundle: No such file or directory
Error: Unable to include the file index.html in the bundle: No such file or directory
```

Deploying from within the directory succeeds:
```
> rsconnect deploy html index.html -n dogfood
Validating server... 	[OK]
Validating app mode... 	[OK]
Making bundle ... 	[OK]
Deploying bundle ... 	[OK]
Saving deployed information... 	[OK]
Building static content...
Launching static content...
Deployment completed successfully.
```

**Comment by kgartland-rstudio** [comment]
We also no do not allow directory deploys for html content on the voila branch:
```
> rsconnect version
1.15.0b2.dev1+g5566262
> rsconnect deploy html bundles/html-single-file/ -n dogfood
Validating server... 	[OK]
Validating app mode... 	[OK]
Making bundle ... 	[ERROR]: Unable to find a valid html entry point.
Error: Unable to find a valid html entry point.
```

The public build deploys successfully:
```
> rsconnect version
1.14.1
> rsconnect deploy html bundles/html-single-file/ -n dogfood
Validating server... 	[OK]
Validating app mode... 	[OK]
Making bundle ... 	[OK]
Deploying bundle ... 	[OK]
Saving deployed information... 	[OK]
Building static content...
Launching static content...
Deployment completed successfully.
```

**Comment by bcwu** [comment]
> We also no do not allow directory deploys for html content on the voila branch:
> 
> ```
> > rsconnect version
> 1.15.0b2.dev1+g5566262
> > rsconnect deploy html bundles/html-single-file/ -n dogfood
> Validating server... 	[OK]
> Validating app mode... 	[OK]
> Making bundle ... 	[ERROR]: Unable to find a valid html entry point.
> Error: Unable to find a valid html entry point.
> ```
> 
> The public build deploys successfully:
> 
> ```
> > rsconnect version
> 1.14.1
> > rsconnect deploy html bundles/html-single-file/ -n dogfood
> Validating server... 	[OK]
> Validating app mode... 	[OK]
> Making bundle ... 	[OK]
> Deploying bundle ... 	[OK]
> Saving deployed information... 	[OK]
> Building static content...
> Launching static content...
> Deployment completed successfully.
> ```

fixed by [e686448](https://github.com/rstudio/rsconnect-python/pull/360/commits/e686448772570e63ee3ec36aade914371a7da3d9)
